src/sysfiles.c

```diff
@@ -3483,15 +3481,6 @@ Obj SyIsDir ( const Char * name )
   else return ObjsChar['?'];
 }
 
-#else
-
-Obj SyIsDir ( const Char * name )
-{
-  return ObjsChar['?'];
```

Comment:
I'd rather get an error on platforms that don't have `lstat`, so that an alternative can be implemented, than silently degrading this functionality.